browser/resources/settings/brave_search_engines_page/brave_search_engines_page.html

```diff
@@ -9,15 +9,18 @@
     </settings-dropdown-menu>
   </div>
 </template>
-<template is="dom-if" if="[[shouldShowSearchSuggestToggle_()]]" restamp>
+<!-- <template is="dom-if" if="[[shouldShowSearchSuggestToggle_()]]" restamp>
   <settings-toggle-button id="searchSuggestToggle"
       class="hr"
       pref="{{prefs.search.suggest_enabled}}"
       label="$i18n{searchSuggestPref}"
       sub-label="$i18n{searchSuggestPrefDesc}">
   </settings-toggle-button>
-</template>
-<if expr="enable_extensions">
+</template> -->
+
+<!-- commented the html structure code for the search engine page of the settings page. -->
+
+<!-- <if expr="enable_extensions">
```

Comment:
if this tag is commented that wouldn't we need to comment the next line as well

Reply:
which tag bhaiya?

Reply:
iske niche wala search-toggle-button

Reply:
didnt able to understand bhaiya, please provide more detail

Reply:
https://meet.google.com/ivy-aspk-ysj